crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -355,6 +366,18 @@ void LateUpdateLods()
             if (_lodDataShadow) _lodDataShadow.UpdateLodData();
         }
 
+        void LateUpdateTiles()
+        {
+            var definitelyUnderwater = ViewerHeightAboveWater < -5f;
+
+            // TODO - dont call this every frame!! save off list from oceanbuilder?
+            var tiles = GetComponentsInChildren<Renderer>();
+            foreach (var tile in tiles)
+            {
+                tile.GetComponent<Renderer>().enabled = (!definitelyUnderwater) || (_viewpoint.position - tile.bounds.ClosestPoint(_viewpoint.position)).magnitude < VolumeExtinctionLength;
```

Comment:
@huwb if you are already getting a list of renderers, is there a reason why GetComponent is being called here as well?

Reply:
Shitty prototyping code reasons :). good catch!

Reply:
Sweet, using the list we already generate for the manual culling :) so sorted.